### DISCUSSION TITLE
Combine documents into one pdf

### DISCUSSION THREAD

**Discussion by markusthiel** [docusealco/docuseal#297] [Q&A]
Is it possible, when multiple documents have been uploaded in a process, to combine them into a single document as a result in the email, instead of sending individual documents?

**Comment by omohokcoj** [comment]
Hi @markusthiel , currently that's not possible - but you can download a combined PDF vie the UI on the /submissions/id page
![image](https://github.com/docusealco/docuseal/assets/5418788/6eefc9ab-db31-4b12-bed9-5dc60125b3f7)

**Reply by markusthiel** [reply]
Thank you! That’s already great. But in my case, I would like to provide a single document to all involved parties. For example, I have documents that are forwarded to three people one after another, and each one needs to sign. They should also be able to upload a single document.

In the end, you receive an email with the individual documents. There is an option to click on a link to download the documents. If there were an option to download a single document, that would be fantastic.

Currently, I can only do it for myself with an account.

**Reply by markusthiel** [reply]
Okay, I found a solution: It's important that all involved signers have an account. Then they can simply click on the link in the final email and will reach the option you mentioned! This works for me for now. However, in the long run, it would be useful if one could choose in the settings whether individual or combined PDFs are sent by email.

**Reply by Tom-H-L** [reply]
It would also be great to have the option to append the Audit Log Document to the Original Document, so to have it all in one file, as e.g. Pandadoc does it by default.

**Reply by AminZoubaa** [reply]
I was also looking for a solution to get the document as a combined signed PDF.
It seems that @omohokcoj  has already extended the API https://github.com/docusealco/docuseal/commit/8aac935641c1c051cc8448d65e7ba39267b20c59 to be able to use the parameter: `include=combined_document_url` for: 
- `https://api.docuseal.eu/submissions/${id}` 

Thank you for that! ❤️ 🤝 

example:
```javascript
  const API_KEY = 'XXX';
  const submission = '12345';

  const document_url = fetch(
    `https://api.docuseal.eu/submissions/${submission}?include=combined_document_url`,
    {
      method: 'GET',
      headers: {
        'X-Auth-Token': API_KEY,
        'content-type': 'application/json',
      },
    }
  )
    .then((res) => {
      return res.json();
    })
    .then((data) => {
      return data.combined_document_url;
    });
```

Maybe the docs needs an update 💪

**Comment by Tekkharibo** [comment]
I reopen it to propose to add this button inside the `/template/id`, I think it's more easier to use it
